translation-tests/src/integration-test/java/eu/europeana/api/translation/tests/web/LangDetectionRestIT.java

```diff
@@ -62,13 +64,22 @@ void langDetectionWithoutLangParam() throws Exception {
   @Test
   void langDetectionMissingTextParam() throws Exception {
     String requestJson = "{}";
-    mockMvc
+    String response = mockMvc
         .perform(
             post(BASE_URL_DETECT)
               .header(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON_VALUE)
               .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
               .content(requestJson))
-        .andExpect(status().isBadRequest());
+        .andExpect(status().isBadRequest())
+        .andReturn().getResponse().getContentAsString();
+    
+    JSONObject obj = new JSONObject(response);
```

Comment:
The jackson ObjectMapper should be used here to parse the response to EuropeanaApiErrorResponse object, using a parseErrorResponse() method